### ISSUE TITLE
AIORTC does not provide access to track information from RemoteDescription when processing the 'track' event

### ISSUE THREAD

**Issue by andrews239** [aiortc/aiortc#258]
If RemoteDescription provides several video(audio) tracks, then there is no way to distinguish one video track from another...

PeerConnection can pass ssrc parameter for easy identification of tracks.
```
--- aiortc/rtcrtpreceiver.py 2019-12-13 17:57:43.983661651 -0800
+++ rtcrtpreceiver.py 2020-01-20 08:55:42.196436845 -0800
@@ -159,9 +159,10 @@
 
 
 class RemoteStreamTrack(MediaStreamTrack):
-    def __init__(self, kind):
+    def __init__(self, kind, ssrc=None):
         super().__init__()
         self.kind = kind
+        self.ssrc = ssrc
         self._queue = asyncio.Queue()
 
     async def recv(self):
--- aiortc/rtcpeerconnection.py 2019-12-13 17:57:43.983661651 -0800
+++ rtcpeerconnection.py 2020-01-20 08:54:37.782570954 -0800
@@ -740,7 +741,7 @@
                     direction in ["recvonly", "sendrecv"]
                     and not transceiver.receiver._track
                 ):
-                    transceiver.receiver._track = RemoteStreamTrack(kind=media.kind)
+                    transceiver.receiver._track = RemoteStreamTrack(kind=media.kind, ssrc=media.ssrc)
                     trackEvents.append(
                         RTCTrackEvent(
                             receiver=transceiver.receiver,

```

A simple use case.
```
            @pc.on("track")
            def on_track(track):
                trackID=track.ssrc[0].msid.split()[0]
                app.data[f"recorder{trackID}"].addTrack(track)
```

**Comment by jlaine** [comment]
I don't think you should be relying on having access to the ssrc, as the standard MediaSteamTrack does not have an ssrc member:

https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack

**Comment by jlaine** [comment]
I just fixed #269 which now gives you access to the track's ID as set by the remote party.
